src/app/components/categories/categories.component.ts

```diff
@@ -4,13 +4,13 @@ import { SpotifyApiService } from 'src/app/services/spotify.service';
 @Component({
   selector: 'app-categories',
   templateUrl: './categories.component.html',
-  styleUrls: ['./categories.component.scss']
+  styleUrls: ['./categories.component.scss'],
 })
 export class CategoriesComponent implements OnInit {
   public categories: any = [];
-  public isLoaded: boolean = false;
+  public isLoaded = false;
```

Comment:
app-loader - maybe would be better add app-loader to the root App component and write service, that will interact with it - switch spinner on/off by calling service methods instead of duplicating template and boolean flags in different components?